flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -34,7 +34,7 @@
 /**
  * Context object with optional arguments for a Flink Scan.
  */
-class ScanContext implements Serializable {
+public class ScanContext implements Serializable {
```

Comment:
Is this change related to this PR ?

Reply:
I'm good with not having to duplicate setters in `FlinkSource` 👍